Tests/MSFT_xIPAddress.Tests.ps1

```diff
@@ -105,7 +127,7 @@ InModuleScope MSFT_xIPAddress {
 
             It 'should be $true' {
                 $Splat = @{
-                    IPAddres = '192.168.0.1'
+                    IPAddress = '192.168.0.1'
                     InterfaceAlias = 'Ethernet'
                 }
                 $Result = ValidateProperties @Splat
```

Comment:
I'd add in a check that your Mocks were actually called here so we know more then just if it ran without failure.